dockerfiles/Dockerfile-coda-demo

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:18.04
+# Run with `docker build --build-arg coda_version=<version>`
+ARG coda_version
+ARG deb_repo=stable
+RUN echo "Building image with version $coda_version"
+
+ADD scripts/run-demo.sh /run-demo.sh
+
+# Dependencies
+RUN apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    strace \
+    dumb-init \
+    libssl1.1 \
+    libprocps6 \
+    libgmp10 \
+    libgomp1 \
+    libffi6 \
+    apt-transport-https \
+    ca-certificates \
+    dnsutils \
+    jq \
+    tzdata && \
+  rm -rf /var/lib/apt/lists/*
+
+# coda package
+RUN echo "deb [trusted=yes] http://packages.o1test.net $deb_repo main" > /etc/apt/sources.list.d/coda.list \
+  && apt-get update \
+  && apt-get install --force-yes coda-testnet-postake-medium-curves=$coda_version -y \
+  && mkdir /root/keys \
+  && chmod go-rwx /root/keys \
+  && echo '{"box_primitive":"xsalsa20poly1305","pw_primitive":"argon2i","nonce":"7S1YA5PinXhnLgLJ3xemVnVPWdJdhKZ9RSNQbns","pwsalt":"AzDoECCYyJL8KuoB2vrsVc9Wg3xJ","pwdiff":[134217728,6],"ciphertext":"5UQuiQVbXPmR63ikri792dWR6Dz5dYZm8dLzwDyqWovdP5CzrLY6Fjw3QTHXA9J3PDkPZpvhrQfGkgU81kr9184dfoJDhn5EXxJMCAM44SZdmBYVszEQaSQnyy4BwsbRXmfjBMSW9ooGu2a5dFi5KHX5na6fr62VUB"}' > /root/keys/demo-block-producer \
+  && chmod go-rwx /root/keys/demo-block-producer \
+  && rm /var/lib/coda/coda_genesis_* \
+  && echo '{"pk":"4vsRCVMNTrCx4NpN6kKTkFKLcFN4vXUP5RB9PqSZe1qsyDs4AW5XeNgAf16WUPRBCakaPiXcxjp6JUpGNQ6fdU977x5LntvxrSg11xrmK6ZDaGSMEGj12dkeEpyKcEpkzcKwYWZ2Yf2vpwQP","balance":100000,"sk":null,"delegate":null}' >> /genesis-accounts.json \
```

Comment:
Should this be `> /genesis-accounts.json`? Not that it should make a difference in this case